testing/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/PaparazziPluginSmokeTest.groovy

```diff
@@ -65,7 +66,17 @@ class PaparazziPluginSmokeTest extends AbstractSmokeTest implements RunnerFactor
             </manifest>""".stripIndent()
 
         expect:
-        agpRunner(agpVersion, 'testDebug').build()
+        agpRunner(agpVersion, 'testDebug')
```

Comment:
Can we not leverage `WithAndroidDeprecations` and reuse `expectIsPropertyDeprecationWarnings`?

Reply:
No, because we need to use `maybeExpectLegacyDeprecationWarning` here. I'll add another method that does use that so we can share between this and santa tracker.